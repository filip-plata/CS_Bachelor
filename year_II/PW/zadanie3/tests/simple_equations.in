3 2 4
1 x[1] = (x[2] + x[2])
2 x[0] = ((-x[1]) *x[1])
3 x[1] -18
