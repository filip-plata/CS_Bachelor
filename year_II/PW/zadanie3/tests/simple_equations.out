1 P
2 P
3 P -324
